chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -1937,3 +1939,74 @@ async def get_coin_records(coin_ids: Collection[bytes32]) -> List[CoinRecord]:
             assert result[1] != MempoolInclusionStatus.FAILED
     except ValidationError as e:
         assert e.code == expected
+
+
+@pytest.mark.anyio
+async def test_fill_rate_block_validation(simulator_and_wallet: OldSimulatorsAndWallets) -> None:
```

Comment:
I think you should make this test deterministic. we have a fixture called `seeded_random` that you can use

Reply:
Done.